### PR TITLE
add isObject helper

### PR DESCRIPTION
**Pull request by tbrandle**

### CODE REVIEW

File: src/isObject.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ * In Javascript, this is a two part check because an Array is technically an Object. isObject() is a shortcut to verify that a data type is an Object Literal.
+ * @example <caption>Verify object contains keys.</caption>
```
Comment (5290charlie) [inline src/isObject.js]: @tbrandle Update the caption to represent what this function is doing. "Verify object exists and is not an array" or something

File: src/isObject.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+ * var { isObject } = require('@bowtie/utils')
+ *
+ * var obj = { a: '1', b: '2', c: '3' }
+ * var arr = [1, 2, 3]
+ *
+ * isObject(obj) // returns true
+ * isObject(arr) // returns false
+ * @param {object, array, string, integer} item - data type to verify
```
Comment (5290charlie) [inline src/isObject.js]: @tbrandle to allow any param type, just use `@param {*} item ...`

File: src/isObject.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ */
+const isObject = (item) => {
+  if (item && item instanceof Object && !Array.isArray(item)) {
```
Comment (5290charlie) [inline src/isObject.js]: @tbrandle can just return the boolean from this if statement :)

File: test/isObject.spec.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+/* eslint-env mocha */
+/* eslint-disable no-unused-expressions */
+
+const expect = require('chai').expect
+const { isObject } = require('../')
+
+const inputs = {
+  obj: {},
+  arr: [],
```
Comment (5290charlie) [inline test/isObject.spec.js:9]: @tbrandle would probably be worth testing a non-empty object & array to verify both work. Might also be worth testing additional variable types (integer, null, undefined, etc) since the signature should allow anything to be checked and return false if not exist or not an object

**Review by 5290charlie — CHANGES_REQUESTED** [review]
